authors/apps/profiles/tests/test_get_profile.py

```diff
@@ -11,13 +11,14 @@ def test_get_all_profiles(self):
         self.client.credentials(HTTP_AUTHORIZATION='Bearer ' + self.token)
         response = self.client.get(self.profile_url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        assert response.data[0]['bio'] == ''
-        assert response.data[0]['image'] == None
+        assert response.data['count'] == 1
+        assert response.data['next'] == None
 
     def test_current_profile(self):
```

Comment:
Try to writing docstring explaining what the test does
```suggestion
    def test_current_profile(self):
        """what the current profile test"""
```